scrape/manager.go

```diff
@@ -167,7 +169,7 @@ func (m *Manager) Run(tsets <-chan map[string][]*targetgroup.Group) error {
 }
 
 func (m *Manager) reloader() {
-	ticker := time.NewTicker(5 * time.Second)
+	ticker := time.NewTicker(time.Duration(m.opts.DiscoveryReloadInterval))
```

Comment:
Can we have a check here? if it is lower than 5s, use 5s as minimum. this will also prevent library users to have bad surprises.

Reply:
I dont know if we want to enforce 5s minimal? 
Are we sure we want to forbid user having every second pod discovery for example ?

Reply:
yes.

Reply:
ok, will do